sierra_adapter/terraform/stack/vhs_sierra_sourcedata.tf

```diff
@@ -1,5 +1,5 @@
 module "vhs_sierra" {
-  source             = "git::github.com/wellcomecollection/terraform-aws-vhs.git//hash-store?ref=v3.4.3"
+  source             = "git::github.com/wellcomecollection/terraform-aws-vhs.git//hash-store?ref=add-name-tags"
```

Comment:
Just approved this, so update and apply when it's built: https://github.com/wellcomecollection/terraform-aws-vhs/pull/14